### DISCUSSION TITLE
Is the documentation up to date?

### DISCUSSION THREAD

**Discussion by maglims** [OfficeDev/microsoft-365-agents-toolkit#11603] [General]
On this [documentation](https://learn.microsoft.com/en-us/javascript/api/@microsoft/teamsfx/teamsusercredential?view=msteams-client-js-latest), it says "TeamsUserCredential can only be used within Teams". Is this accurate?

The reason we are asking is because there are some developers trying to use TeamsUserCredential for SSO on other hosts(like outlook mobile) and run into issues. We would like to check if it is ever supported on other platforms, or the document needs to be updated.

Thanks!

**Comment by SLdragon** [comment]
Hi, @maglims , 
TeamsUserCredential can works within Teams Desktop/Mobile/Web, Outlook Desktop/Mobile/Web, for mobile platform, due to webview inside different device may have some limitations, so it may have issues. We will update our document, and feel free to create issue if you have some problems when using it~
